README.md

```diff
@@ -30,6 +30,8 @@ cookiecutter gh:ITISFoundation/cookiecutter-osparc-service
 
 Check [FAQ](./FAQ.md)
 
+## Tutorials and guides
+A video tutorial is available [here](https://youtu.be/yoKXHMQcs1Y?si=lmJgDM-Y52jvwS3d).
```

Comment:
You could do something more "fancy" thatn "here" and embedd the video?

SEE https://www.darenmay.com/custommayd/developer/training/videos/markdown/2020/06/23/embed-youtube-video-in-markdown.html